src/transformers/trainer.py

```diff
@@ -1253,18 +1253,35 @@ def torch_jit_model_eval(self, model, dataloader, training=False):
                 return model
             jit_inputs = []
             example_batch = next(iter(dataloader))
+            if "labels" in example_batch:
```

Comment:
This change doesn't make sense. there is no relationship between label, label_smotherer and jit.trace.

Reply:
this logic is copied from trainer.py. if label smoother option is opened. label should be pop from input.